account_financial_risk/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_partner_risk_exceeded_wiz_user,Partner Risk Exceeded Wizard (Internal user),model_partner_risk_exceeded_wiz,base.group_user,1,0,0,0
```

Comment:
```suggestion
access_partner_risk_exceeded_wiz_user,Partner Risk Exceeded Wizard (Internal user),model_partner_risk_exceeded_wiz,base.group_user,1,1,1,1
```

You need more permission to the wizard, this must be fixed in order to raise the wizard

Reply:
> Attend @alan196 request changes



> Attend @alan196 request changes

Sure! Changes are done.